batch/batch/cloud/gcp/driver/zones.py

```diff
@@ -11,6 +11,42 @@
 
 log = logging.getLogger('zones')
 
+MACHINE_FAMILY_VALID_ZONES = {
```

Comment:
Is there a way to scrape this information automatically? if not, then I'd like to  (1) make an issue to figure out how to do this automatically and (2) remove the "n1" hard coding. Otherwise, we're going to break the Australian's deployment.

Reply:
Yeah, there's API calls to get all the machine types:

https://cloud.google.com/compute/docs/reference/rest/v1/machineTypes/aggregatedList?hl=en

which includes a zone field and a list of accelerators.  We can update the code to use that.